x/custom/msg.go

```diff
@@ -0,0 +1,32 @@
+package custom
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/migration"
+)
+
+func init() {
+	migration.MustRegister(1, &CreateCustomStateMsg{}, migration.NoModification)
+}
+
+const (
+	pathCreateCustomStateMsg = "custom/create_custom_state"
```

Comment:
Some time ago we have updated [how handler for a message is registered](https://github.com/iov-one/weave/issues/817). Thanks to this change you no longer have to create a const with a path name. You can embed it directly in the `Path` method as it is the only place where it is used.

This change might require recent weave version.

Reply:
Will do this change right away.

Reply:
Yeah, we should upgrade to weave 0.18 (or 0.19 if ready)